### PR TITLE
Add Istio lifecycle mgmt sample resources

### PR DESCRIPTION
**Pull request by Rachael-Graham**
Adds some updated example install files for Gloo Mesh, including IstioLifecycleOperator files, new multicluster options for 2.1, and new single-cluster install files for 2.1.

### CODE REVIEW

File: getting-started/2.1/getting-started.yaml [hunk 1]
```diff
@@ -0,0 +1,405 @@
+###########################################################################################
```
Comment (Rachael-Graham) [inline getting-started/2.1/getting-started.yaml:1]: Putting this in for now, but no equivalent docs yet. Will test more thoroughly when I work on the single-cluster install path for GS.

File: helm-install/2.1/single-cluster.yaml [hunk 2]
```diff
@@ -0,0 +1,394 @@
+###########################################################################################
```
Comment (Rachael-Graham) [inline helm-install/2.1/single-cluster.yaml:1]: Putting this in for now, but no equivalent docs yet. Will test more thoroughly when I work on the single-cluster install path for prod-level setup.

File: istio-install/gm-managed/gm-istiod-openshift.yaml [hunk 3]
```diff
@@ -0,0 +1,189 @@
+apiVersion: admin.gloo.solo.io/v2
```
Comment (Rachael-Graham) [inline istio-install/gm-managed/gm-istiod-openshift.yaml:1]: New lifecycle manager example files. I did not put these in a versioned file, as I believe these will be more fluid/easy to make small adjustments to from version to version. Thoughts?

File: helm-install/2.1/single-cluster.yaml [hunk 4]
```diff
@@ -0,0 +1,394 @@
...
+leaderElection: true
+
+# License settings
+# Your Gloo Mesh Enterprise license that you got from your Solo account representative
+glooMeshLicenseKey: $GLOO_MESH_LICENSE_KEY
+# Provide your $GLOO_GATEWAY_LICENSE_KEY to also install Gloo Gateway
+glooGatewayLicenseKey: ""
+# Provide your $GLOO_NETWORK_LICENSE_KEY to also install Gloo Gateway
+glooNetworkLicenseKey: ""
+# To provide license keys in a secret in the cluster instead, specify the secret name.
```
Comment (artberger) [inline helm-install/2.1/single-cluster.yaml:30]: Does this secret have to be in a particular namespace and cluster, like `gloo-mesh` on management cluster?

File: istio-install/gm-managed/gm-istiod-openshift.yaml (hunk 3 above)
Comment (artberger) [reply istio-install/gm-managed/gm-istiod-openshift.yaml:1]: Sounds good; we can try it and if it becomes too versioned, split it out.

File: helm-install/2.1/single-cluster.yaml (hunk 4 above)
Comment (Rachael-Graham) [reply helm-install/2.1/single-cluster.yaml:30]: yes ty!

```suggestion
# To instead provide license keys in a secret in the gloo-mesh namespace of the management cluster, specify the secret name.
```

**Pull request merged**